brave/brave-core
# Removes auto contribute for non verified publishers

Resolves https://github.com/brave/brave-browser/issues/2803

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
plan 1
- enable rewards
- visit some verified and non-verified sites
- make sure that you only see verified sites in ac table
- wait for reconcile to happen
- make sure that you only send contribution to verified sites

plan 2
- go to master branch
- enable rewards
- visit some verified and non-verified sites
- make sure that you see verified and non-verified sites in ac table
- close browser
- switch to this PR branch
- go to settings page
- make sure that you only see verified sites in ac table
- wait for reconcile to happen
- make sure that you only send contribution to verified sites


## Reviewer Checklist:
